web/ui/react-app/src/Graph.tsx

```diff
@@ -28,11 +37,20 @@ interface GraphProps {
     resolution: number;
   } | null;
 }
+interface GraphState {
+  selectedSeriesIndex: number;
+  hoveredSeriesIndex: number;
```

Comment:
I see hover and single-series selection features implemented here. The old UI also had the feature to turn off selected series (this is what the checkmark symbol was for in the old UI). Were you still planning on adding that? I wonder if people actually use it... in case we decide to not add that feature, I don't think we need the checkmarks.

Reply:
Hm, I didn't notice the "off" state on checkbox click. I thought is related to the "selected" state of the row, but yes, ok, I'll added it if you think it is usable.

Reply:
Hm yeah, I'm also not sure whether we still need it. How about we omit that feature for now, and we can still add it later if users really ask for it. Less features is always good when they aren't used :)

But then I think we don't need the checkmark icons at all for now.